### PR TITLE
Getting setup for debugging

### PR DESCRIPTION
**Pull request by Roxy-3D**
I wasn't going to check anything in until I got stuff running...   But with all the random edits happening in the areas I'm working...   It is best to merge the code very often.

### CODE REVIEW

File: Marlin/ubl_G29.cpp [hunk 1]
```diff
@@ -1615,22 +1615,22 @@ double zzz[] = { 0.05, 0.05, 0.15, 0.15 };
...
 
       results = lsf_linear_fit(xxx9, yyy9, zzz9, nine_size);
-SERIAL_PROTOCOLPAIR("\nxxx9->A =", results->A); 
-SERIAL_PROTOCOLPAIR("\nxxx9->B =", results->B); 
-SERIAL_PROTOCOLPAIR("\nxxx9->D =", results->D); 
-SERIAL_PROTOCOL("\n");
+SERIAL_ECHOPAIR("\nxxx9->A =", results->A); 
+SERIAL_ECHOPAIR("\nxxx9->B =", results->B); 
+SERIAL_ECHOPAIR("\nxxx9->D =", results->D); 
+SERIAL_ECHO("\n");
```
Comment (thinkyhead) [inline Marlin/ubl_G29.cpp]: `SERIAL_EOL` is preferred (prints char, not string) for single newline. `SERIAL_CHAR` for other single characters. And `SERIAL_ECHOPGM("...")` should be used for literal strings, to ensure storage in progmem.

Comment (Bob-the-Kuhn) [reply Marlin/ubl_G29.cpp]: You've got my curiosity up.  Why is there both a SERIAL_ECHO and a SERIAL_PROTOCOL series?

**Comment by Roxy-3D** [comment]
More updates coming soon...    Things should be neat and tidy by the end of the day.   But I wasted a lot of time yesterday because other untested changes were doing bad things in the command parsing.

**Comment by thinkyhead** [comment]
That was my bad. I didn't know you used the parameters even if command parsing fails and returns an error. I assumed that if an error was returned from parameter parsing then G29 would bail at that point.

**Pull request merged**